app/views/representations/_resource.html.slim

```diff
@@ -7,7 +7,7 @@ li.list-item
     span.sr-only Caption:
     = to_html resource.title
     - representations.each do |representation|
-      .timeline-item
+      div class="timeline-item representation-#{representation.id}"
```

Comment:
Two notes, one trivial and one important.

Trivially, you can write this as:

```
.timeline-item class="representation-#{representation.id}"
```

More importantly, this should not be a class name, but an id, e.g. `.timeline-item id="representation-#{representation.id}"`